### PR TITLE
Feature/create common test component

### PR DESCRIPTION
**Pull request by tanyushanel**
CPT2107201417-1877x757

### CODE REVIEW

**Comment by RomPavel** [comment]
In general, I don't like when we mix outdated code with new one. 
If we go with one route for all steps, lets remove specific routes for steps

we have 'Submit' buttons in each step, if we go with one route we should update selectedTab and not change redirect to different route

File: exadel-trainee-app/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -86,6 +93,10 @@ const routes: Routes = [
     path: Route.grammarTest,
     component: GrammarTestComponent,
   },
+  {
+    path: Route.commonTest,
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/app-routing.module.ts:97]: Why do you need this

File: exadel-trainee-app/src/app/models/common-test/common-test.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  templateUrl: './common-test.component.html',
+  styleUrls: ['./common-test.component.scss'],
+})
+export class CommonTestComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  ngOnInit(): void {}
+
+  nextTest(): void {
+    // if (this.router.url === `/${Route.grammarTest}`) {
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/common-test/common-test.component.ts:16]: this code will be never used

File: exadel-trainee-app/src/app/models/grammar-test/grammar-test.component.ts [hunk 3]
```diff
@@ -6,5 +6,7 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./grammar-test.component.scss'],
 })
 export class GrammarTestComponent implements OnInit {
+  selectedIndex = 0;
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/grammar-test/grammar-test.component.ts:9]: what the purpose of it

File: exadel-trainee-app/src/app/app-routing.module.ts (hunk 1 above)
Comment (tanyushanel) [reply exadel-trainee-app/src/app/app-routing.module.ts:74]: We had path: Route.grammarTest, I deleted it, now we have path: Route.commonTest instead to navigate by clickind btn from user-profile.

Comment (RomPavel) [reply exadel-trainee-app/src/app/app-routing.module.ts:74]: Then what is the purpose of path 'commonTest' on 24 line?

Comment (tanyushanel) [reply exadel-trainee-app/src/app/app-routing.module.ts:74]: Innattentive. Forgot about lazyLoaded commonTest.

File: exadel-trainee-app/src/app/models/user-profile/user-profile.component.ts [hunk 4]
```diff
@@ -25,7 +25,7 @@ export class UserProfileComponent implements OnInit {
 
   onStartButtonClick(): void {
     this.router.navigateByUrl('/', { skipLocationChange: true }).then(() => {
-      this.router.navigate([Route.grammarTest]);
+      this.router.navigate(['commonTest']);
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/user-profile/user-profile.component.ts:28]: Enum please

**Review by RomPavel — APPROVED** [review]

File: exadel-trainee-app/src/app/models/common-test/common-test-routing.module.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { GrammarTestComponent } from '../grammar-test/grammar-test.component';
+import { ListeningTestComponent } from '../listening-test/listening-test.component';
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/common-test/common-test-routing.module.ts:4]: this are not needed

**Review by RomPavel — APPROVED** [review]

**Pull request merged**